aten/src/ATen/native/sparse/SparseTensor.cpp

```diff
@@ -303,8 +305,16 @@ SparseTensor dense_to_sparse(const Tensor& self, int64_t sparse_dim){
     indices = indices.contiguous();  // many sparse CUDA kernels require contiguity, see issue #12633
   }
 
-  std::vector<Tensor> ix = indices.chunk(indices.size(0), 0);
-  Tensor values = self.index(ix).squeeze(0).clone();
+  Tensor values;
+  if (self.dim() > 0) {
+    std::vector<Tensor> ix = indices.chunk(indices.size(0), 0);
+    values = self.index(ix).squeeze(0).clone();
```

Comment:
It's kind of surprising this code doesn't generalize to the `self.dim() == 0` case. Where exactly does it fail? `index()` with an empty vector?

Reply:
Yes.

Reply:
well even before I don't believe you can chunk empty tensors, and that's consistent with numpy.

Reply:
I guess you could go one way or another, but chunking an empty tensor seems like a perfectly well defined thing to do :)

Reply:
ya, if that would have fixed the issue I would have looked into it more, but considering neither call worked, it didn't seem worth it.